### ISSUE TITLE
Relay crashes when new client joins

### ISSUE THREAD

**Issue by Pamoi** [dedis/prifi#82]
With error message `F : (                protocols.(*PriFiSDAProtocol).SetConfig: 155) - Less than one client reachable (I'm a relay, and there's no use starting the protocol) !`

To reproduce run:

```
./prifi.sh relay
./prifi.sh trustee 0
./prifi.sh client 0
./prifi.sh client 1
```

It seems that no clients at all are present in the `Identities` field of the `*PriFiSDAWrapperConfig` struct which is passed to the `PriFiSDAProtocol` by the service (in `setConfigToPriFiProtocol`).

**Comment by lbarman** [comment]
working on it. Meanwhile, check out ba57b195cd3c216f0910d931cd7792a4a1f65529 if you need to work, I think the bug was added around f8d236fb381187d0443db860645421a864f26c54. Apologies!

**Comment by lbarman** [comment]
Fixed ef0f7f41910f3a70215a29124683ca0085aa2df4
